### PR TITLE
Introduce pipeline pause/unpause API v1 #4076

### PR DESCRIPTION
**Pull request by GaneshSPatil**

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/pipeline_operation_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+      allow(@pipeline_pause_service).to receive(:pause).with(@pipeline_name, @pause_cause, @user, anything) do |pipeline_name, pause_cause, user, result|
+        result.setMessage(LocalizedMessage.string('PIPELINE_PAUSE_SUCCESSFUL', pipeline_name));
+      end
+
+      post_with_api_header :pause, pipeline_name: @pipeline_name, pause_cause: @pause_cause
+      expect(response).to be_ok
+    end
+
+    it 'should show errors occurred while pausing the pipeline' do
```
Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/pipeline_operation_controller_spec.rb]: Should move to a different describe block? Add route tests?

File: server/webapp/localize.xml [hunk 2]
```diff
@@ -794,4 +794,8 @@
     <entry key="CAN_NOT_UPDATE_RESOURCES_ON_ELASTIC_AGENT">Resources on elastic agents with uuids [{0}] can not be updated.</entry>
     <entry key="PENDING_AGENT_INVALID_OPERATION">Pending agents [{0}] must be explicitly enabled or disabled when performing any operations on them</entry>
     <entry key="NO_OPERATION_PERFORMED_ON_AGENTS">No Operation performed on agents.</entry>
+    <entry key="PIPELINE_PAUSE_SUCCESSFUL">Pipeline ''{0}'' paused successfully.</entry>
```
Comment (varshavaradarajan) [inline server/webapp/localize.xml]: `localize_ja.xml` hasn't been deleted, so, you need to add these entries there too?

Comment (arvindsv) [reply server/webapp/localize.xml]: Shouldn't we delete `localize_ja.xml`?

Comment (varshavaradarajan) [reply server/webapp/localize.xml]: Yes. I remember @GaneshSPatil and me talking about it. I suppose another PR for it? @ketan wanted to get rid of even localize.xml, I think

Comment (ketan) [reply server/webapp/localize.xml]: > Shouldn't we delete localize_ja.xml?

I vote for you for president...

**Pull request closed without merging**